float64[] az_list
float64[] el_list
float64 start_time
---
bool success
string error_msg
